Dockerfile

```diff
@@ -9,7 +9,7 @@ RUN zypper -n install \
         python lttng-ust-devel babeltrace-devel \
         librados2 python2-pylint python3-pylint \
         bash vim tmux git aaa_base ccache wget jq google-opensans-fonts \
-        oath-toolkit python-devel python-Cython python-PrettyTable psmisc \
+        liboath-devel python-devel python-Cython python-PrettyTable psmisc \
```

Comment:
I think we should remove this package from installation. Ceph will install the correct package when running `install-deps`.

Reply:
Fixed, thanks for pointing this out.